docs/advanced/symbol-server-compatibility.md

```diff
@@ -312,6 +329,7 @@ The debug id is in all cases lowercase in hex format and computed as follows:
 
 - **PE**: `<Signature><Age>` (age in hex, not padded)
 - **PDB**: `<Signature><Age>` (age in hex, not padded)
+- **Portable PDB**: `<Signature>FFFFFFFF`
```

Comment:
I believe for the unified layout we actually use the full debug id, can you double-check? It should be consistent what symsorter would do in that case, which might also be a good idea to check